### PR TITLE
grpcurl 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by crewmanmud**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

New formula for [grpcurl](https://github.com/fullstorydev/grpcurl) utility.  Have refreshed branch, updated utility version after #34641 closed by bot.

fullstorydev/grpcurl#52

### CODE REVIEW

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Thanks for the pull request!

File: Formula/grpcurl.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    (buildpath/"src/github.com/fullstorydev/grpcurl").install buildpath.children
+    cd "src/github.com/fullstorydev/grpcurl/cmd" do
+      system "go", "build", "-ldflags", "-X main.version=#{version}",
+             "-o", bin/"grpcurl", "./..."
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/grpcurl", "-help"
```
[the remark this answers is not in the document]
Comment (crewmanmud) [reply Formula/grpcurl.rb]: Sure, I'll create a protobuf message and attempt to describe it with the utility.

**Comment by crewmanmud** [comment]
@SMillerDev I've addressed the changes you asked for, let me know if all good. 👍

**Review by SMillerDev — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @crewmanmud

File: Formula/grpcurl.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  url "https://github.com/fullstorydev/grpcurl/archive/v1.1.0.tar.gz"
+  sha256 "8c8af1643c4d549947a2f2198f5c21d634b2a24b31de975a5784841a9127c82f"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    (buildpath/"src/github.com/fullstorydev/grpcurl").install buildpath.children
+    cd "src/github.com/fullstorydev/grpcurl/cmd" do
+      system "go", "build", "-ldflags", "-X main.version=#{version}",
```
Comment (jhump) [inline Formula/grpcurl.rb]: @crewmanmud, where does the value for `#{version}` come from? I don't see anything else in this file to indicate that it is v1.1.0. Does the `Formula` base class know how to parse it from the `url` field?

Comment (SMillerDev) [reply Formula/grpcurl.rb]: Yes it does, it's pretty neat

**Pull request closed without merging**